pyhmmer/easel.pyx

```diff
@@ -5170,32 +5170,32 @@ cdef class DigitalSequence(Sequence):
         Translate the sequence using the given genetic code.
 
         Arguments:
-            genetic_code (`pyhmmer.easel.GeneticCode`): The genetic code to 
-                use for translating the sequence. If none provided, the 
+            genetic_code (`pyhmmer.easel.GeneticCode`): The genetic code to
+                use for translating the sequence. If none provided, the
                 default uses the standard translation table (1) and expects
                 DNA sequences.
 
         Returns:
-            `pyhmmer.easel.DigitalSequence`: The translation of the 
+            `pyhmmer.easel.DigitalSequence`: The translation of the
             input sequence, in digital mode. The name of the source
-            sequence will be stored in the ``source`` attribute of 
+            sequence will be stored in the ``source`` attribute of
             the result.
```

Comment:
You can remove the second sentence, as the name of the sequence will be stored in the `name` attribute.